gitbucket.war 3.2 library is a duplicate

gitbucket.war includes c3p0-0.9.5.jar and c3p0-0.9.1.1.jar.

It fails to start tomcat.
OS:CentOS
Tomcat:8.0.18 
java:version "1.8.0_45"

When removing the c3p0-0.9.1.1.jar, to succeed to start.

```
29-Jun-2015 11:09:08.003 SEVERE [localhost-startStop-1] org.apache.catalina.core.StandardContext.listenerStop クラス org.scalatra.servv
let.ScalatraListener のリスナインスタンスにコンテキスト破棄イベントを送信中の例外です
 java.lang.NoClassDefFoundError: Could not initialize class gitbucket.core.servlet.Database$
        at gitbucket.core.servlet.InitializeListener.contextInitialized(InitializeListener.scala:30)
        at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4720)
        at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5154)
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
        at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:725)
        at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:701)
        at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:714)
        at org.apache.catalina.startup.HostConfig.deployWAR(HostConfig.java:917)
        at org.apache.catalina.startup.HostConfig$DeployWar.run(HostConfig.java:1701)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.reflect.UndeclaredThrowableException
        at com.sun.proxy.$Proxy4.find(Unknown Source)
        at com.mchange.v2.cfg.MConfig.readVmConfig(MConfig.java:75)
        at com.mchange.v2.c3p0.cfg.C3P0Config.findLibraryMultiPropertiesConfig(C3P0Config.java:157)
        at com.mchange.v2.c3p0.cfg.C3P0Config.<clinit>(C3P0Config.java:143)
        ... 21 more
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at com.mchange.v1.lang.Synchronizer$1.invoke(Synchronizer.java:58)
        ... 25 more
Caused by: java.lang.NoSuchMethodError: com.mchange.v2.cfg.BasicMultiPropertiesConfig.<init>([Ljava/lang/String;Ljava/util/List;)V
        at com.mchange.v2.cfg.ConfigUtils.read(ConfigUtils.java:63)
        at com.mchange.v2.cfg.MConfig$CSManager.recreateFromKey(MConfig.java:153)
        at com.mchange.v1.cachedstore.NoCleanupCachedStore.find(NoCleanupCachedStore.java:63)
        ... 30 more
```

--- a/project/build.scala
+++ b/project/build.scala
@@ -65,7 +65,7 @@ object MyBuild extends Build {
       "com.mchange" % "c3p0" % "0.9.5",
       "com.typesafe" % "config" % "1.2.1",
       "com.typesafe.akka" %% "akka-actor" % "2.3.10",
-      "com.enragedginger" %% "akka-quartz-scheduler" % "1.3.0-akka-2.3.x"
+      "com.enragedginger" %% "akka-quartz-scheduler" % "1.3.0-akka-2.3.x"  exclude("c3p0","c3p0")
     ),
     play.twirl.sbt.Import.TwirlKeys.templateImports += "gitbucket.core._",
     EclipseKeys.withSource := true,

